src/oscar/apps/checkout/surcharges.py

```diff
@@ -0,0 +1,52 @@
+from decimal import Decimal as D
```

Comment:
I think we should have unit tests for each of the built-in charge classes below - percentage and flat - to verify that their output is what we would expect.

Reply:
Yea I agree. Ik won't be able to do it today. I hope to do this in the upcoming week.

Reply:
I added tests in tests/integration/checkout/test_surcharges.py